client/webserver/site/src/js/wallets.ts

```diff
@@ -722,7 +722,11 @@ export default class WalletsPage extends BasePage {
       Doc.hide(this.page.showChangePW)
       this.changeWalletPW = false
       this.setPWSettingViz(false)
-    } else Doc.show(this.page.showChangePW)
+      Doc.hide(this.reconfigForm.fileSelector)
+    } else {
+      Doc.show(this.page.showChangePW)
```

Comment:
You can do `Doc.show(this.page.showChangePW, this.reconfigForm.fileSelector)`.

Reply:
Good point.  @vctt94 if you combine them like this, we can merge the PR now.